www/strf03/cv08/signup.php

```diff
@@ -0,0 +1,56 @@
+<?php
+session_start();
+
+require 'db.php';
+$submittedForm = $_SERVER['REQUEST_METHOD'] == 'POST';
+// $submittedForm = !empty($_POST);
+
+if ($submittedForm) {
+    $email = $_POST['email'];
+    $password = $_POST['password'];
+
+    if (!filter_var($email, FILTER_VALIDATE_EMAIL)) {
+        $errors['email'] = 'Please use a valid email';
+    }
+
+    if (strlen($password) < 3) {
```

Comment:
mininálně 8 se doporučuje

Reply:
jojo, bylo to pouze zkušební, abych nemusel vždycky zadávat dlouhé heslo.